homework-3/phrase.py

```diff
@@ -0,0 +1,14 @@
+from collections import Counter
+
+
+def generate_phrase(characters, phrase):

```

Comment:
This solution looks exactly like the one shown here:
https://stackoverflow.com/a/73198472

Reply:
Is there another way you can solve it without using an existing solution?